### PR TITLE
Fixed wording per https://github.com/docker/docker.github.io/issues/7281

### PR DESCRIPTION
**Pull request by ahh-docker**
Signed-off-by: Anne Henmi <anne.henmi@docker.com>

**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by ahh-docker** [comment]
@andrewhsu PTAL.

**Comment by ahh-docker** [comment]
Related to https://github.com/docker/docker.github.io/issues/7281

**Comment by ahh-docker** [comment]
PR is related here: https://github.com/docker/docker.github.io/issues/7281

File: docs/reference/commandline/node_promote.md [hunk 1]
```diff
@@ -26,8 +26,8 @@ Options:
 
 ## Description
 
-Promotes a node to manager. This command targets a docker engine that is a
-manager in the swarm.
+Promotes a node to manager. This command targets a docker engine that is not a manager
```
Comment (thaJeztah) [inline docs/reference/commandline/node_promote.md]: Hm... I think the confusing here is that;

- This command must be run against an engine that _is_ a manager (only managers can promote/demote a node)
- But the _node_ to promote is not yet a manager

Perhaps something like (better suggestions welcome 😅);

```Markdown
This command can only be executed on a manager node.
```

**Comment by ahh-docker** [comment]
@thaJeztah PTAL. Merge if it's ready.

**Comment by thaJeztah** [comment]
@ahh-docker can you squash your commits, and make sure it has a sign-off? Otherwise CI will fail

**Comment by ahh-docker** [comment]
@thaJeztah I think I did it. Can you check and see if it's gtg?

**Comment by thaJeztah** [comment]
Hmm. It think something went wrong, as I see a merge commit in there, and four commits total.

Let me try describe the steps;

Checkout your branch

```bash
git checkout promote-node-7281
```


Make sure your remotes are up-to-date (in my case the "upstream" points to `docker/cli`, and "origin" points to my fork (`thaJeztah/cli`));

```bash
git fetch upstream -v; git fetch origin -v; 
```


Start an interactive rebase against the "upstream master"; the "merge" commit should already dissappear, but to combine the changes of the other commits, we'll have to "squash" them;

```bash
git rebase -i upstream/master
```

Doing the above should open your editor with something like below:

```bash
pick f58964ed Fixed wording per https://github.com/docker/docker.github.io/issues/7281
pick c0f6aab4 Fixed wording per https://github.com/docker/docker.github.io/issues/7281
pick 581b7684 Update node_promote.md

# Rebase 6deb4f1f..150c3294 onto 6deb4f1f (3 commands)
#
# Commands:
# p, pick = use commit
# r, reword = use commit, but edit the commit message
# e, edit = use commit, but stop for amending
# s, squash = use commit, but meld into previous commit
# f, fixup = like "squash", but discard this commit's log message
# x, exec = run command (the rest of the line) using shell
# d, drop = remove commit
#
# These lines can be re-ordered; they are executed from top to bottom.
#
# If you remove a line here THAT COMMIT WILL BE LOST.
#
# However, if you remove everything, the rebase will be aborted.
#
# Note that empty commits are commented out
```


Looks like the first commit actually has a merge conclict, so we'll drop that one (which can be done either by changing `pick` to `drop`, or by removing the line)
and you want to squash the last two commits so edit the last line and change `pick` to `squash` (or `s`);

```
drop f58964ed Fixed wording per https://github.com/docker/docker.github.io/issues/7281
pick c0f6aab4 Fixed wording per https://github.com/docker/docker.github.io/issues/7281
squash 581b7684 Update node_promote.md
```

Once you save the file, and exit your editor, another file is opened, allowing you to update the commit message of the combined commits;

```
# This is a combination of 2 commits.
# This is the 1st commit message:

Fixed wording per https://github.com/docker/docker.github.io/issues/7281

Signed-off-by: Anne Henmi <anne.henmi@docker.com>

# This is the commit message #2:

Update node_promote.md

```

You could change it to, e.g.

```
docs: clarify that node promote/demote must be executed from a manager

Fixed wording per https://github.com/docker/docker.github.io/issues/7281

Signed-off-by: Anne Henmi <anne.henmi@docker.com>
```


After you saved the description, the rebase/squash should be succesful, now you need to _force_ push your changes to update the PR (assuming the remote pointing to your fork is named "origin");


```
git push -f origin
```

**Comment by ahh-docker** [comment]
@thaJeztah 

So I got this far:
```
$ git checkout promote-node-7281
Already on 'promote-node-7281'
Your branch is up to date with 'origin/promote-node-7281'.
$ git fetch upstream -v; git fetch origin -v; 
fatal: 'upstream' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
From github.com:ahh-docker/cli
 = [up to date]        promote-node-7281       -> origin/promote-node-7281
 = [up to date]        18.06                   -> origin/18.06
 = [up to date]        18.09                   -> origin/18.09
 = [up to date]        extra_greater_than_7220 -> origin/extra_greater_than_7220
 = [up to date]        master                  -> origin/master
AH-MBP13-2017:/Users/annehenmi/repos/ahh-fork-cli- git remote
origin
upstreamengcli
$ git remote -v
origin	git@github.com:ahh-docker/cli.git (fetch)
origin	git@github.com:ahh-docker/cli.git (push)
upstreamengcli	git@github.com:docker/cli.git (fetch)
upstreamengcli	git@github.com:docker/cli.git (push)
$ git rebase -i upstream/master
fatal: Needed a single revision
invalid upstream 'upstream/master'
```

Not sure what's going on, but I'll see if I can get some help fixing it.

**Comment by ahh-docker** [comment]
@thaJeztah Ok, I went through the steps and fixed it. Let me know if there's anything else it needs. PTAL again. Sorry.

**Comment by thaJeztah** [comment]
```
$ git remote -v
origin	git@github.com:ahh-docker/cli.git (fetch)
origin	git@github.com:ahh-docker/cli.git (push)
upstreamengcli	git@github.com:docker/cli.git (fetch)
upstreamengcli	git@github.com:docker/cli.git (push)
```

Ah, yes, that's why I usually recommend to use consistent names for your remotes, then you can work in any repository and always use the same commands (e.g.`upstream` for the upstream repository, and `origin` to reference your own fork of the repository)

**Pull request merged**